serial/tools/list_ports_windows.py

```diff
@@ -21,6 +21,7 @@
 from ctypes.wintypes import ULONG
 from ctypes.wintypes import HKEY
 from ctypes.wintypes import BYTE
+import winerror
```

Comment:
does not seem to be in the standard library

Reply:
Thanks for pointing this out - you were correct. This was a module provided by `pywin32` that I (incorrectly) thought was part of the standard library on windows. I removed the dependency and added the error as a file-level constant similar to other windows errors handled here.